src/common/locales/en/translation.json

```diff
@@ -0,0 +1,28 @@
+{
+  "CHECKING_DEPS": "Checking dependencies",
```

Comment:
We should avoid using underscores in keys because i18next [uses underscores for plurals](https://www.i18next.com/translation-function/plurals#singular-plural) and context.

I think we should use lower camel case for key names. This what i18next also mostly uses in their docs.

Reply:
True, i guess i should have read the docs a bit more

Reply:
They have a 20min video where they go over pretty much all features :)

Reply:
Guess i should watch that then